quilt/tools/command.py

```diff
@@ -372,41 +381,92 @@ def _push_package(dry_run=False):
             }
         )
 
-    _push_package(dry_run=True)
+    print("Fetching upload URLs from the registry...")
```

Comment:
TMI? Is this here because it could take a long time and we want to give feedback to the user?